deployment/constructor_params/mainnet/root.yml

```
@@ -0,0 +1,54 @@
+deployment:
+  name: mainnet-root
+  chain_id: 1 # Ethereum Mainnet
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: mainnet.json
+
+constants:
+  # Threshold Network - References:
+  # - https://docs.threshold.network/resources/contract-addresses/mainnet/threshold-dao
+  T_TOKEN_ETH_MAINNET: "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5"
+  T_STAKING_CONTRACT: "0x01B67b1194C75264d06F808A921228a95C765dd7"
+  THRESHOLD_COUNCIL_ETH_MAINNET: "0x9F6e831c8F8939DC0C830C6e492e7cEf4f9C2F5f"
+
+  # FxPortal addresses - References:
+  # - https://github.com/0xPolygon/fx-portal#deployment-addresses
+  # - https://github.com/0xPolygon/fx-portal/blob/main/config/config.json
+  # - https://wiki.polygon.technology/docs/pos/design/bridge/l1-l2-communication/state-transfer/#prerequisites
+  FXPORTAL_CHECKPOINT_MANAGER: "0x86e4dc95c7fbdbf52e33d563bbdb00823894c287"
+  FXPORTAL_FXROOT: "0xfe5e5D361b2ad62c541bAb87C45a0B9B018389a2"
+  POLYGON_CHILD: "invalid" # Temporary invalid value that will be overwritten by workaround in deployment script
+
+  # TACo specific constants:
+  IN_SECONDS_1_HOUR: 3600
+  IN_SECONDS_1_DAY: 86400
+  IN_SECONDS_7_DAYS: 604800
+  IN_SECONDS_60_DAYS: 5184000
+  IN_SECONDS_182_DAYS: 15724800
+  IN_SECONDS_364_DAYS: 31449600
+  FORTY_THOUSAND_TOKENS_IN_WEI_UNITS: 40000000000000000000000
+  TIMESTAMP_FOR_2024_01_01_0000_UTC: 1704067200  # FIXME: Made up value, waiting for https://github.com/nucypher/nucypher/issues/3342
```

```suggestion
  TIMESTAMP_FOR_2024_01_01_0000_UTC: 1703930400
```

Based on https://github.com/nucypher/nucypher/issues/3342#issuecomment-1821205103

December 30th at midnight UTC-11

---

```suggestion
  TIMESTAMP_FOR_2023_12_31_1100_UTC: 1703930400  # FIXME: Made up value, waiting for https://github.com/nucypher/nucypher/issues/3342
```
technically someone in the kiribati islands can game this and commit one hour into january 1st 2024, but honestly good for them

---

FYI
```
In [4]: m = maya.MayaDT(epoch=1703930400)

In [5]: m.rfc3339()
Out[5]: '2023-12-30T10:00:00.0Z'
```